cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -48,6 +50,15 @@ enum {
 };
 
 /**
+ * @brief   Available peripheral buses
+ */
+enum {
+    AHB1,           /**< AHB1 bus */
+    AHB2,           /**< AHB2 bus */
+    AHB3            /**< AHB3 bus */
+};
```

Comment:
I'd move a change here to a separate PR and use it in all other stm32s too then.